modules/social_features/social_group/social_group.module

```diff
@@ -317,10 +317,20 @@ function social_group_preprocess_group(array &$variables) {
       }
     }
   }
-
   // Count number of group members.
-  $group_members_count = \Drupal::service('social_group.group_members_count');
-  $variables['group_members'] = $group_members_count->getGroupMemberCount($group);
+  $variables['group_members'] = count($group->getMembers());
+
+  // Prepare variables for statistic block.
+  if ($group->getGroupType()->hasContentPlugin('group_node:event')) {
+    $variables['group_events'] = count($group->getContent('group_node:event'));
+  }
+  if ($group->getGroupType()->hasContentPlugin('group_node:topic')) {
+    $variables['group_topics'] = count($group->getContent('group_node:topic'));
+  }
+  if ($group->getGroupType()->id() !== 'closed_group' || $group->getMember($account) || $group->hasPermission('manage all groups', $account)) {
+    $variables['group_stream_url'] = Url::fromUserInput('/group/' . $group->id() . '/stream');
```

Comment:
We shouldn't be using `fromUserInput` but instead use the routename with the proper argument. This ensures that we don't create issues if the URL is changed.

Reply:
@Kingdutch
Fixed for all cases in this module.